src/main/java/seedu/address/ui/EventListPanel.java

```diff
@@ -29,26 +31,60 @@
     @FXML
     private ListView<EventCard> eventListView;
 
+    private Label emptyLabel = new Label("Event List is empty!");
+
     public EventListPanel(ObservableList<Activity> eventList) {
         super(FXML);
         setConnections(eventList);
         registerAsAnEventHandler(this);
+        setUpPlaceHolder();
+        //maybe do not need this
+        //eventListView.managedProperty().bind(eventListView.visibleProperty());
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.